state/user.go

```diff
@@ -37,12 +37,14 @@ func userIDFromGlobalKey(key string) string {
 }
 
 func (st *State) checkUserExists(name string) (bool, error) {
+	nameToLower := strings.ToLower(name)
```

Comment:
Personal preference.. but I don't like variables that sound like they're about actions in the future, when they're about the past. imo we're representing a `nameLowerCase` rather than a `nameToLower`.

Reply:
Yeah agreed, there was already code see addUser which named it this way. So I was going for consistency. I don't mind changing them all.

Reply:
It looks like the variables in those private methods were [created 4 years ago](https://github.com/juju/juju/commit/6c720319d758d216921bd2ea29865d8e80a7f3d2). We probably won't hurt anyone's feelings by changing them now.

Maybe add to PR?